packages/styles/components/_c.toggle.scss

```diff
@@ -104,6 +108,8 @@
   &--hide-label {
     #{$parent}__content {
       @include set-hidden-input();
+
+      padding-left: 0;
```

Comment:
The `padding-left: 0;` is not necessary here, as there is already a `padding: 0;` in the mixin `set-hidden-input()`.